airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_35_46_001__DefaultConnectionName.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import static org.jooq.impl.DSL.asterisk;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class V0_35_46_001__DefaultConnectionName extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_35_46_001__DefaultConnectionName.class);
+
+  public static void defaultConnectionName(final DSLContext ctx) {
+    LOGGER.info("updating connection name column");
+    final Field<UUID> id = DSL.field("id", SQLDataType.UUID.nullable(false));
+    final Field<String> name = DSL.field("name", SQLDataType.VARCHAR(256).nullable(false));
+    List<Connection> connections = getConnections(ctx);
+
+    for (final Connection connection : connections) {
+      final Actor sourceActor = getActor(connection.getSourceId(), ctx);
+      final Actor destinationActor = getActor(connection.getDestinationId(), ctx);
+      final String connectionName = sourceActor.getName() + " -> " + destinationActor.getName();
+
+      ctx.update(DSL.table("connection"))
+          .set(name, connectionName)
+          .where(id.eq(connection.getConnectionId()))
```

Comment:
what about adding `WHERE name == 'default'` that way if any names are set we don't override them. i kinda don't think any are set differently.